### PR TITLE
add details on na plugin status

### PR DESCRIPTION
**Pull request by carns**

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -133,24 +133,32 @@ are used internally. There should not be any need for using them directly.
...
-  local node communication. Remote memory access is done through
-  cross-memory attach ([CMA][cma]) on Linux. See this
-  [page]({% post_url 2017-01-30-shared-memory %}) for design details
-  about that plugin (Transparent usage of this plugin has not been implemented yet).
-* _CCI_: flexible connection and disconnection. Provides support for TCP, UDP,
-  verbs, uGNI and shared memory. Plugin shows good performance and supports
-  InfiniBand as well as Cray<sup>®</sup> Gemini/Aries interconnect but is more
-  experimental.
+* _BMI_: The BMI library itself is no longer under active feature development
+  beyond basic maintenance, but the the BMI Mercury NA plugin provides a very stable
```
Comment (soumagne) [inline documentation.md]: typo :)  the BMI Mercury

File: documentation.md [hunk 2]
```diff
@@ -133,24 +133,32 @@ are used internally. There should not be any need for using them directly.
...
+* _SM_: This is the integrated shared memory NA plugin included with Mercury.  Plugin is stable and shows good performance, but it is only applicable local node communication.  The eventual goal of this plugin is to provide a transparent shortcut for other NA plugins when they connect to local services, but it is also useful as a primary transport for single-node services. *Technical notes*:
+    * Supports dynamic client connection and disconnection.
+    * Low CPU consumption (e.g., idles without busy spinning or using threads)
+    * RMA (for Mercury bulk operations) is implemented natively through cross-memory attach ([CMA][cma]) on Linux, and there are fallback methods for other platforms as well. See this
+  [page]({% post_url 2017-01-30-shared-memory %}) for design details.
+  * _CCI_: This is the most mature NA plugin available for general purpose use, but some transport plugins within CCI are more robust than others.  *Technical notes*:
+    * Low CPU consumption (e.g., idles without busy spinning or using threads), with the exception of the TCP CCI plugin, see below.
```
Comment (soumagne) [inline documentation.md:155]: I would add a note here though saying that the CCI plugins create threads internally though.

**Review by soumagne — COMMENTED** [review]
Looks good to me, thanks!

File: documentation.md (hunk 2 above)
Comment (carns) [reply documentation.md:155]: Thanks Jerome, I pushed another commit to address your comments.

**Pull request merged**